ibm/mas_devops/roles/kubeturbo/README.md

```diff
@@ -0,0 +1,8 @@
+Kubeturbo
+============
+Installs a kubeturbo instance and sends data to a defined Turbonomic instance under the cluster name.
+
```

Comment:
readme needs filling out with the options that are required and optional and what those mean etc.

Reply:
I had the required and optional setup in the wrong branch - gh-pages.  I will add them to the one here.